Playground/src/components/hamburgerMenu.tsx

```diff
@@ -95,6 +96,7 @@ export class HamburgerMenuComponent extends React.Component<IHamburgerMenuCompon
         var engineOptions = [
             {
                 label: "WebGL2",
+                tooltip: "WebGL 2.0",

```

Comment:
maybe use the longer text here as well (like earlier)

Reply:
yes sorry! fixed